src/main/java/duke/TaskAssigner.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Events;
+import duke.tasks.Task;
+import duke.tasks.ToDos;
+
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.ArrayList;
+
+public class TaskAssigner {
+
+    public static ArrayList<String> task_t = new ArrayList<>(List.of("todo", "event", "deadline"));
+
+    public TaskAssigner() {}
+
+    public Task assignTask(String command) throws DukeException {
```

Comment:
I like that your function name is written beginning with a verb! It helps to convey the action which the function is performing.